### PR TITLE
Upgrade to Asciidoctor 1.6.x

### PR DESCRIPTION
**Pull request by gsmet**
Hi @binout ,

It turns out that Asciidoctor 1.6 changed a bit how the initialization is done. So while the new core artifact should solve the issue of a minor upgrade, we still need a branch for the 1.6.x upgrade.

I did the work to upgrade to 1.6 (the PR is against master but it should be put in a specific branch).

Do you think we could merge this to a 1.6.x branch and release a parallel 1.6.0-alpha.3 release while you're at it?

It would be greatly appreciated!

Thanks!

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -59,10 +59,10 @@
         <maven.apache-rat-plugin.version>0.7</maven.apache-rat-plugin.version>
         <maven.jacoco-maven-plugin.version>0.7.2.201409121644</maven.jacoco-maven-plugin.version>
 
-        <asciidoctorj.version>1.5.4.1</asciidoctorj.version>
-        <asciidoctorj-pdf.version>1.5.0-alpha.11</asciidoctorj-pdf.version>
+        <asciidoctorj.version>1.6.0-alpha.3</asciidoctorj.version>
+        <asciidoctorj-pdf.version>1.5.0-alpha.15</asciidoctorj-pdf.version>
         <ant.version>1.8.0</ant.version>
-        <jruby.version>1.7.25</jruby.version>
+        <jruby.version>9.1.8.0</jruby.version>
```
Comment (gsmet) [inline pom.xml]: The version is aligned with the version used by asciidoctorj-pdf which was the most recent.

File: src/test/java/org/asciidoctor/ant/extensions/TwitterMacro.java [hunk 2]
```diff
@@ -15,28 +15,32 @@
...
-    protected Object process(AbstractBlock parent, String twitterHandle, Map<String, Object> attributes) {
+    public Object process(ContentNode parent, String twitterHandle, Map<String, Object> attributes) {
         // Define options for an 'anchor' element.
         Map<String, Object> options = new HashMap<String, Object>();
         options.put("type",":link");
         options.put("target", "http://www.twitter.com/@"+twitterHandle);
 
         // Prepend twitterHandle with @ as text link.
-        final Inline inlineTwitterLink = createInline(parent, "anchor", "@"+twitterHandle, attributes, options);
+        final PhraseNode inlineTwitterLink = createPhraseNode(parent, "anchor", "@"+twitterHandle, attributes, options);
```
Comment (gsmet) [inline src/test/java/org/asciidoctor/ant/extensions/TwitterMacro.java]: I based these changes on the ManpageMacro present in the asciidoctorj repository.

See https://github.com/asciidoctor/asciidoctorj/blob/asciidoctorj-1.6.0/asciidoctorj-core/src/test/java/org/asciidoctor/extension/ManpageMacro.java .

**Comment by gsmet** [comment]
Asciidoctorj does not support Java 1.6 anymore. i force pushed the removal of Java 6 support in the .travis.yml file.

**Comment by binout** [comment]
Thx for this great work !
Ok for the 1.6.x branch and the release 1.6.0-alpha.3

**Comment by gsmet** [comment]
Force pushed a rebase after the latest merges.

**Comment by binout** [comment]
Merged on 1.6.x branch

**Comment by binout** [comment]
Release 1.6.0-alpha3 done

**Comment by gsmet** [comment]
Thanks!

**Pull request closed without merging**